documentation/docs/guidedtour.md

```diff
@@ -83,22 +83,31 @@ Copy the sources of the application into your own Git repository. While we will
 
 1. Provide a name for your new item (for example, *My First Pipeline*) and select **Multibranch Pipeline**.
    <p align="center">
-   ![Create Pipeline Job](images/JenkinsNewItemPipeline-1.png "Jenkins New Item")
+   ![Create Pipeline Job](images/JenkinsNewItemPipeline.png "Jenkins New Item")
    </p>
+   **Note:** The ready-made Continuous Delivery pipelines of project "Piper" require to run as **Multibranch Pipeline**.
 
-1. For **SCM**, choose **Git**.
-
-1. For **Repository URL** in the **Repositories** section, enter the URL of your Git repository, for example `https://github.com/<your-org>/cloud-cf-helloworld-nodejs`. **Note:** If your repository is protected, you must provide your credentials in the **Credentials** section.
+1. For **Branch Sources**, choose **Add source**, and select the **Git** as source repository.
```

Comment:
"the **Git** repository as source" oder "**Git** as source repository" (ohne "and")